src/main/webapp/views/viewCard.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
```

Comment:
Lots of this <head> looks repeated across files, will probably be the case for all frontend html files you make. Is there any way to make one <head> that is inserted/used as needed by each file? (Just so you don't have to keep adding all these scripts etc each time)

Reply:
Agree with you, I think @RicardoChapaRomero  will make that changes once the navbar is finished.